PowerEditor/src/WinControls/TaskList/TaskList.h

```diff
@@ -69,10 +70,13 @@ class TaskList : public Window
 		return (((TaskList *)(::GetWindowLongPtr(hwnd, GWLP_USERDATA)))->runProc(hwnd, Message, wParam, lParam));
 	};
 
+	void GetCurrentDisplayResolution();

```

Comment:
I'd expect a `get...Resolution` function to *return* the resolution i.e. both horizontal and vertical.